openghg/util/_met.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np
+
+
+def _get_site_pressure(inlet_heights: list, site_height: float) -> list[float]:
+    """Calculate the pressure levels required
+
+    Args:
+        inlet_height: Height(s) of inlets
+        site_height: Height of site
+    Returns:
+        list: List of pressures
+    """
+    import re
+
+    if not isinstance(inlet_heights, list):
+        inlet_heights = [inlet_heights]
+
+    measured_pressure = []
+    for h in inlet_heights:
+        try:
+            # Extract the number from the inlet height str using regex
+            inlet = float(re.findall(r"\d+(?:\.\d+)?", h)[0])
+            measurement_height = inlet + float(site_height)
+            # Calculate the pressure
+            pressure = float(1000 * np.exp((-1 * measurement_height) / 7640))
```

Comment:
Would be good to split the numbers in this calculation into variables (especially `7640`) - just so it's easier in future to know where these values came from.